larcorealg/Geometry/details/OpDetGeoSpecFillers.h

```diff
@@ -0,0 +1,178 @@
+/**
+ * @file   larcorealg/Geometry/details/OpDetGeoSpecFillers.h
+ * @brief  Definition of internal data of `geo::OpDetGeo`.
+ * @author Gianluca Petrillo (petrillo@slac.stanford.edu)
+ * @see    larcorealg/Geometry/details/OpDetGeoSpecFillers.cxx
+ * 
+ */
+
+#ifndef LARCOREALG_GEOMETRY_DETAILS_OPDETGEOSPECFILLERS_H
+#define LARCOREALG_GEOMETRY_DETAILS_OPDETGEOSPECFILLERS_H
+
+// LArSoft libraries
+#include "larcorealg/Geometry/details/OpDetGeoSpecs.h"
+#include "larcorealg/Geometry/Decomposer.h" // geo::PlaneBase<>
+#include "larcorealg/Geometry/LocalTransformationGeo.h"
+#include "larcoreobj/SimpleTypesAndConstants/geo_vectors.h" // geo::Point_t...
+
+// Framework libraries
+#include "cetlib_except/exception.h"
+
+#if 0
+// class header
+#include "larcorealg/Geometry/OpDetGeo.h"
+
+// LArSoft libraries
+#include "larcorealg/Geometry/geo_vectors_utils.h" // geo::vect::makeFromCoords()
+#include "larcorealg/CoreUtils/RealComparisons.h"
+#include "larcoreobj/SimpleTypesAndConstants/PhysicalConstants.h" // util::pi()
+
+
+// C/C++ standard libraries
+#include <cmath>
+#endif // 0
```

Comment:
Remove disabled code.

Reply:
Sorry, Gianluca.  Working on a review and prematurely submitted this.  The comment stands, though.

Reply:
Yup, I left something behind. Fixed.